projects/angular/src/layout/nav/aria-current-link.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2016-2022 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Directive, HostBinding, OnDestroy, OnInit } from '@angular/core';
+import { RouterLinkActive } from '@angular/router';
+import { Subscription } from 'rxjs';
+
+@Directive({ selector: '[clr-aria-current-link]', host: { '[attr.aria-current]': 'stepAriaCurrent' } })
+export class ClrAriaCurrentLink implements OnInit, OnDestroy {
+  @HostBinding('attr.aria-current') ariaCurrent = undefined;
+
+  private subscriptions: Subscription[] = [];
+
+  constructor(private rla: RouterLinkActive) {}
+
+  ngOnInit() {
+    this.subscriptions.push(this.listenForRouterLinkChange());
+  }
+
+  listenForRouterLinkChange() {
```

Comment:
We can have this subscription directly in the `this.subscriptions.push` as this is the only subscription in the class.